src/main/scala/org/constellation/datastore/SnapshotTrigger.scala

```diff
@@ -49,7 +49,7 @@ class SnapshotTrigger(periodSeconds: Int = 5)(implicit dao: DAO, cluster: Cluste
           case Right(created) =>
             resetNodeState(stateSet)
               .flatMap(_ => dao.metrics.incrementMetricAsync[IO](Metrics.snapshotAttempt + Metrics.success))
-              .flatMap(_ => dao.redownloadService.persistOwnSnapshot(created.height, created.hash))
+//              .flatMap(_ => dao.redownloadService.persistLocalSnapshot(created))
```

Comment:
Maybe we should remove it instead of commenting out? Otherwise please provide the comment why it is commented. It helps distinguishing such things from accidental leftovers